gems/redis/4.2/redis.rbs

```diff
@@ -198,8 +198,9 @@ class Redis
   # @return [String] `PONG`
   def ping: (?String? message) -> String
 
-  def zcard: (String|Symbol key) -> Integer
-  def zadd: (String|Symbol key, *untyped args, ?nx: bool, ?xx: bool, ?lt: bool, ?gt: bool, ?ch: bool, ?incr: bool) -> [bool, Integer, Float]
-  def zincrby: (String|Symbol key, Integer|Float increment, String member) -> Float
-  def zrem: (String|Symbol key, String|Array[String] member) -> [bool, Integer]
+  def zcard: (_ToS key) -> Integer
+  def zadd: (_ToS key, *untyped args, ?nx: bool, ?ch: bool, ?incr: bool) -> (bool | Integer | Float)
+          | (_ToS key, *untyped args, ?xx: bool, ?ch: bool, ?incr: bool) -> (bool | Integer | Float)
+  def zincrby: (_ToS key, Integer|Float increment, String member) -> Float
```

Comment:
`#zincrby` seems to [accept `"-inf"` and `"+inf"`](https://github.com/redis/redis-rb/blob/v4.2.5/test/lint/sorted_sets.rb#L145-L149), so how about adding these literals for `increment`?

```suggestion
  def zincrby: (_ToS key, Integer|Float|"-inf"|"+inf" increment, String member) -> Float
```

Reply:
thanks suggestion!